app/assets/stylesheets/thredded/components/_select2.scss

```diff
@@ -0,0 +1,38 @@
+.thredded--main-container .select2-container {
+  width: 100%;
+
+  .select2-choices {
+    border: $form-border;
+    padding: $small-spacing;
+    box-shadow: $form-box-shadow;
+    background-image: none;
+
+    > .select2-search-choice {
+      margin: 0 6px 0 0;
+    }
+
+    > .select2-search-field {
+      > input[type="text"] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 5